FactorioBlueprintStringRenderer/src/com/demod/fbsr/EntityRendererFactory.java

```diff
@@ -151,8 +154,7 @@ public static void registerFactories(Consumer<EntityRendererFactory> register, F
 					register.accept(factory);
 				} catch (Exception e) {
 					prototype.debugPrint();
-					System.err.println("FACTORY CLASS: " + factoryClassName);
-					e.printStackTrace();
+					LOGGER.error("FACTORY CLASS: {}", factoryClassName, e);

```

Comment:
In places where there was e.printStackTrace(), I removed it and added the exception as the final parameter to LOGGER.error(). One subtle thing here is that slf4j has support for exception logging, and when used this way, there is one more parameter passed to the varargs than there are placeholders in the template string. And the exception is always the last parameter.